projects/components/src/overlay/sheet/sheet-overlay.component.scss

```diff
@@ -62,3 +43,17 @@
     }
   }
 }
+
+.attached-trigger {
+  cursor: pointer;
+  background: white;
+  width: 200px;
```

Comment:
Should we avoid this? I know it works for our current use case.

Reply:
Let's keep this for now. I want to put some restrictions on the content.